test/index.js

```diff
@@ -129,6 +129,58 @@ describe('t', function () {
     expect(instance.t('header.sign_in')).to.equal('Sign In');
   });
 
+  it('supports custom replace implementation', function () {
+    var instance = new Polyglot({
+      phrases: phrases,
+      replace: function (interpolationRegex, callback) {
+        var phrase = this;
+        var i = 0;
+        var matches = Array.from(phrase.matchAll(interpolationRegex));
```

Comment:
`matchAll` isn't guaranteed to exist in all the engines we support; please add the `string.prototype.matchall` package for this - and neither is `Array.from`, so we'll need to add `array.from` also.

Reply:
Ah sure! I wasn't aware that the tests run in all supported engines but makes sense. Will add those tomorrow!